pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -22,14 +22,14 @@ const (
 	durationFiveMinutes           = "5 minutes"
 )
 
-func NewServiceMonitorCR(namespace string, monitorNamespace string, insecureSkipVerify bool) *v1.ServiceMonitor {
+func NewServiceMonitorCR(namespace string, insecureSkipVerify bool) *v1.ServiceMonitor {
```

Comment:
previously we had separation from the monitorNamespace (the namespace this service monitor is created in) and the namespace that is matched on this service monitor. Now only a single namespace represents both. This seems like it will break existing installations where the monitoring namespace is separate from the kubevirt install namespace.

Reply:
Hey David,
why would it break existing installation?
as far as i know the monitoring namespace in Openshift Virtualization is `openshift-monitoring`
and the installation namespace in there is `openshift-cnv` which is separated.